packages/vats/src/lib-chainStorage.js

```diff
@@ -68,9 +68,6 @@ export function makeChainStorageRoot(
         assert.typeof(value, 'string');
         handleStorageMessage({ key: path, method: 'set', value });
       },
-      clearValue() {
-        handleStorageMessage({ key: path, method: 'set' });
-      },
```

Comment:
This is a hard pill to swallow because there ought to be a conceptual difference between "set to empty" vs. "unset", but factually that is not the case in vstorage.go. So I do not object.